src/xo-app/home/index.js

```diff
@@ -532,79 +532,85 @@ export default class Home extends Component {
     const isAdmin = user && user.permission === 'admin'
 
     if (!props.areObjectsFetched) {
-      return <CenterPanel>
-        <h2><img src='assets/loading.svg' /></h2>
-      </CenterPanel>
+      return <Page>
```

Comment:
`Page` should not be used only to set the title of the page, especially not for the default title.

We should use `DocumentTitle` directly in the root component.